### ISSUE TITLE
Pytorch 實作 Word2Vec with Softmax - seed9D's blog

### ISSUE THREAD

**Issue by seed9D** [seed9D/seed9D.github.io#11]
https://seed9d.github.io/Pytorch-Implement-Naive-Word2Vec-with-Softmax/#more 

 用  pytorch  實現最簡單版本的  CBOW 與 skipgram，objective function 採用 minimize negative log likelihood with softmax
